run.sh

```diff
@@ -76,6 +79,9 @@ else
   printf "Already exists!\n"
 fi
 
+# Enable or Disable debug mode
```

Comment:
On first run yes. But if you stop the container and start it again with the ENV variable, the `wp-cli.yml` won't get generated a second time. But I like your idea of a script I think